bin/sl-svc-install.js

```diff
@@ -11,30 +13,19 @@ if (!opts.command && opts._.length > 0) {
 delete opts._;
 
 if (opts.help || opts.h) {
-  return usage(process.argv[1], console.log.bind(console));
+  return g.t('sl-service-install.txt',
+    process.argv[1], console.log.bind(console));
```

Comment:
.txt contents is loaded as one message with the key being the file name.  g.t('sl-serv-ce-install.txt, ...) is equivalent to g.t('usage: {0} [options] {{-- <app and args>}}\n\nOptions:\n{{-h,--help}} ....